### DISCUSSION TITLE
allow scheduler to call an url with the triggered message

### DISCUSSION THREAD

**Discussion by fkarakas** [etf1/kafka-message-scheduler#15] [Ideas]
Today the scheduler role is to publish a message to a target topic at a specific time in the future. What can be interesting is to be able to call an url with the payload of the message.

Specs:
a new header could be added to the message to specify the url and the method
ex:
scheduler-target-url: http://myapp/channel/online/1
scheduler-target-url-method: POST

or 
scheduler-target-url: POST http://myapp/channel/online/1

**Comment by fkarakas** [comment]
@iotafr @eko @gusmin ?

**Comment by eko** [comment]
Hi, why not, I think it could be interesting in some use cases to have such a HTTP callback mechanism
